CMakeLists.txt

```diff
@@ -60,7 +60,7 @@ include(ROCMCreatePackage)
 include(CheckCXXCompilerFlag)
 include(ROCMHeaderWrapper)
 
-
+set(MIOPEN_ENABLE_AI_HEUR On CACHE BOOL "Enable AI based heuristics")
```

Comment:
How?

Reply:
Pls look how `message(FATAL_ERROR ...` is used in this file.

Reply:
What I did to work around this was if(MIOPEN_EMBED_BUILD) was set to true, then I wouldn't declare the MIOPEN_ENABLE_AI_TUNING_HEUR.

Reply:
But if `-DMIOPEN_ENABLE_AI_KERNEL_TUNING=On` is passed to CMake, then it will be defined regardless of what you did. Please detect case when both MIOPEN_EMBED_BUILD and MIOPEN_ENABLE_AI_KERNEL_TUNING are On and issue the fatal error message.

Reply:
Done.

Reply:
Resolved.